tests/introspection/tests.py

```diff
@@ -79,6 +80,11 @@ def test_unmanaged_through_model(self):
         tables = connection.introspection.django_table_names()
         self.assertNotIn(ArticleReporter._meta.db_table, tables)
 
+    @skipUnlessDBFeature("supports_comments", "supports_foreign_keys")
+    def test_unmanaged_db_comment_model(self):
+        tables = connection.introspection.django_table_names()
+        self.assertNotIn(ArticleReporterWithDBComment._meta.db_table, tables)
```

Comment:
I'm not sure why adding `db_table_comment` would change Django's behavior for unmanaged models :thinking: This looks unnecessary.

Reply:
Removed.